From 29fef51d533d7143696d78de6d7097e9a6a2d8f5 Mon Sep 17 00:00:00 2001
From: Jeroen Hofstee <jhofstee@victronenergy.com>
Date: Mon, 15 May 2023 11:22:51 +0200
Subject: [PATCH] start flashmq for testing purposes

Flashmq is intended as a faster replacement for mosquitto, but if WIP.
Start both services for the time being, so it can be tested.
---
 src/application.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/application.cpp b/src/application.cpp
index 2c5de7d..7018797 100644
--- a/src/application.cpp
+++ b/src/application.cpp
@@ -264,6 +264,9 @@ void Application::manageDaemontoolsServices()
 	mMqttLocalInsecure = mSettings->root()->itemGetOrCreate("Settings/Services/MqttLocalInsecure");
 	mMqttLocalInsecure->getValueAndChanges(this, SLOT(mqttLocalInsecureChanged(QVariant)));
 
+	// Start flashmq as well for testing purposes.
+	new DaemonToolsService(mSettings, "/service/flashmq", "Settings/Services/MqttLocalInsecure", this);
+
 	// MQTT on LAN
 	mMqttLocal = mSettings->root()->itemGetOrCreate("Settings/Services/MqttLocal");
 	mMqttLocal->getValueAndChanges(this, SLOT(mqttLocalChanged(QVariant)));
-- 
2.34.1

